client-mr/core/src/main/java/org/apache/hadoop/mapreduce/v2/app/RssMRAppMaster.java

```diff
@@ -153,9 +141,13 @@ public Thread newThread(Runnable r) {
                 }
               });
 
-      JobConf extraConf = new JobConf();
+      // set loadDefaults to false, rss_conf.xml should only contain conf of RSS,
+      // Hadoop conf is not necessary.
+      Configuration extraConf = new JobConf(false);
```

Comment:
set `loadDefaults` to false, make rss_conf.xml has only `mapreduce.rss.*` properties